central/scannerdefinitions/handler/handler.go

```diff
@@ -92,54 +102,67 @@ type requestedUpdater struct {
 	lastRequestedTime time.Time
 }
 
+// manifest represents the manifest.json file
+// containing Scanner V4 related metadata.
 type manifest struct {
 	Version string `json:"version"`
 }
 
 // httpHandler handles HTTP GET and POST requests for vulnerability data.
 type httpHandler struct {
-	online        bool
-	interval      time.Duration
-	lock          sync.Mutex
-	updaters      map[string]*requestedUpdater
-	onlineVulnDir string
-	blobStore     blob.Datastore
+	// online indicates if we are in online or offline mode.
+	online bool
+	// interval specifies the time period between subsequent updates, in online-mode.
+	interval time.Duration
+	// lock protects updaters.
+	lock sync.Mutex
+	// updaters stores the various updaters which may be required.
+	updaters map[string]*requestedUpdater
+	// rootDir is the root directory into which all data is downloaded.
+	rootDir string
+	// uploadPath is the file path to which "offline data" is stored.
+	// This file will be under the rootDir directory.
+	uploadPath string
+	// blobStore provides access to the blob storage which stores the uploaded "offline data".
+	blobStore blob.Datastore
```

Comment:
The comments make the relationship between these two confusing. There is a directory under the "data directory (aka. root directory)" where offline data is saved. But, a blob store also provides access to a store for the same data (offline data). How are they related? Is `uploadPath` used to initialize the `blobStore`?